octobot_tentacles_manager/configuration/config_file.py

```diff
@@ -0,0 +1,32 @@
+#  Drakkar-Software OctoBot-Tentacles-Manager
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+import json
+import aiofiles
+from os.path import exists
+
+from octobot_commons.config_manager import dump_json
+
+
+async def read_config(config_file: str, raise_exception=False) -> dict:
```

Comment:
I don't understand why you need this param : `raise_exception`. You just have to catch it at call, why not ?

Reply:
It was like this in the previous impl but after a check it turns out it's not used, I'll remove it